lib/cocoapods/installer/user_project_integrator/target_integrator.rb

```diff
@@ -44,6 +44,7 @@ def integrate!
               update_to_cocoapods_0_34,
               update_to_cocoapods_0_37_1,
               update_to_cocoapods_0_39,
+              update_to_cocoapods_0_40,
```

Comment:
So then this could be collapsed with the previous migrator, depending in which release this will happen.